front/src/app/components/forms/convention/SignButton.tsx

```diff
@@ -41,6 +42,7 @@ export const SignButton = ({
   onConfirmClick,
   submitButtonId,
   className,
+  onBeforeOpenSignModal,
```

Comment:
```suggestion
  onOpenSignModal,
```
Parce qu'en vrai, vu que c'est asynchrone, on ne peut pas garantir que ça se passe avant.